### ISSUE TITLE
Normalize speed-based loss function

### ISSUE THREAD

**Issue by Nate711** [jietan/puppersim#20]
It's hard to compare training runs when the clipping value for the maximum speed changes. Maybe we can normalize the loss against the maximum value.
